code/modules/surgery/tools.dm

```diff
@@ -187,7 +187,7 @@
 
 /obj/item/circular_saw/Initialize()
 	. = ..()
-	AddComponent(/datum/component/butchering, 40 * toolspeed, 100, 5, 'sound/weapons/circsawhit.ogg') //saws are very accurate and fast at butchering
+	AddComponent(/datum/component/butchering, 40 * toolspeed, 100, 5, 'sound/effects/circularsaw.ogg') //saws are very accurate and fast at butchering

```

Comment:
to celowe? nie ma takiego pliku chyba?

Reply:
Powinien być, to był nowy dźwięk piłowania kości w trakcie operacji - w sensie `effects/circularsaw.ogg` - o ile dobrze pamiętam. Sprawdzę jeszcze zaraz, bo się doprowadzić do porządku muszę.

Reply:
Sprawdziłem, ścieżka jest poprawna.